source/georef/georef.h

```diff
@@ -82,10 +82,11 @@ struct Vertex {
 
 struct Edge {
     nt::idx_t way_idx = nt::invalid_idx; //< indexe vers le nom de rue
+    nt::idx_t geom_idx = nt::invalid_idx; // geometry index
     navitia::time_duration duration = {}; // duration of the edge
 
     template<class Archive> void serialize(Archive & ar, const unsigned int) {
-        ar & way_idx & duration;
+        ar & way_idx & geom_idx & duration;
```

Comment:
You need to increment ```Data::data_version``` in type/data.cpp
Without this an old kraken can try to load a new dataset, in most case it will lead to a segmentation fault